libs/sqlite3/Makefile

```diff
@@ -99,13 +103,17 @@ TARGET_CFLAGS += \
 	$(if $(CONFIG_SQLITE3_BATCH_ATOMIC_WRITE),-DSQLITE_ENABLE_BATCH_ATOMIC_WRITE) \
 	$(if $(CONFIG_SQLITE3_COLUMN_METADATA),-DSQLITE_ENABLE_COLUMN_METADATA)
 
-CONFIGURE_ARGS += \
+CONFIGURE_ARGS := \
```

Comment:
Does default configuration arguments somehow conflict with this program?

Reply:
Yes. `configure` doesn't accept `target`.

Reply:
@1715173329 See also my comments in https://github.com/openwrt/packages/pull/26344 - sqlite upstream changed how `configure` works.

Reply:
maybe change it to this:
```
CONFIGURE_ARGS := \
	$(filter-out --target=% $(DISABLE_NLS) $(DISABLE_IPV6),$(CONFIGURE_ARGS)) \
```

`--disable-dependency-tracking` does nothing for the new configure script.

Reply:
This looks much better, thanks. I tested this, but without `$(DISABLE_IPV6)`. When would it be set?

Reply:
see rules.mk

Reply:
Fount it. Makes sense.